habitat-lab/habitat/config/benchmark/rearrange/cat_place.yaml

```diff
@@ -42,12 +42,16 @@ habitat:
              height: 256
```

Comment:
Do we want to start training with the same aspect ratio?

Reply:
Ok let me post another update for that and then we can merge this